document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/RelationOperator.java

```diff
@@ -1,6 +1,6 @@
 package org.hypertrace.core.documentstore.expression.operators;
 
-public enum RelationalOperator {
+public enum RelationOperator {
```

Comment:
I think `RelationalOperator` is more easily understandable.
Refer terminology from https://www.geeksforgeeks.org/operators-in-java/.